### ISSUE TITLE
[Feature Request] Read in the SAS file without first converting it to CSV; row-by-row processing capability

### ISSUE THREAD

**Issue by xiaodaigh** [BioStatMatt/sas7bdat.parso#3]
Is there plan to put this feature into the product? I would be keen to see it. It's been a long time since I touched Java but am happy to collaborate.

**Comment by BioStatMatt** [comment]
Yes, it's possible, of course. I hadn't written much Java before this, so we're in the same boat!

**Comment by xiaodaigh** [comment]
BTW I have written a short post regarding your package

http://blog.analytixware.com/2015/02/reading-sas-into-r.html

On 17 February 2015 at 05:02, Matt Shotwell notifications@github.com
wrote:

> Yes, it's possible, of course. I hadn't written much Java before this, so
> we're in the same boat!
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/BioStatMatt/sas7bdat.parso/issues/3#issuecomment-74548981
> .

## 

ZJ

zhuojia.dai@gmail.com

**Comment by harrysouthworth** [comment]
I've found problems with the Parso csv writer - it doesn't escape quotes inside strings. A friend of mine tells me he'll update my Java wrapper so that it works with SQLite rather than csv, so you might want to keep an eye on my unsas repo

**Comment by BioStatMatt** [comment]
Uh oh.  OK. Thanks for the information.

On Tue, Feb 24, 2015 at 11:05 AM, Harry Southworth <notifications@github.com

**Comment by BioStatMatt** [comment]
Great. Thanks for that.

**Comment by printsev** [comment]
Hi Harry, thanks for reporting this issue - we will fix it in the next release. We hope to deliver it in a month or so.

@BioStatMatt - we will try to help you with the original request after the new release.

**Comment by BioStatMatt** [comment]
I'll keep an eye out for next release.

**Comment by printsev** [comment]
Hi Harry, hopefully this issue if fixed in 2.0-SNAPSHOT (https://github.com/epam/parso). It would be great if you could test it using your data set and get back to us. Please feel free to create an issue in parso tracker.
